src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -58,8 +76,11 @@
   private String description;
   private String frameworkName;
   private String slavesUser;
-  private String principal;
-  private String secret;
+  private String credentialsId;
+  @Deprecated
```

Comment:
We cannot deprecate these without giving a heads up to existing users. Can you please email our mailing list and get feedback about this change?